presto-docs/src/main/sphinx/release/release-325.rst

```diff
@@ -2,6 +2,9 @@
 Release 325
 ===========
 
+* Created: 2019-11-14
+* Source code tag: `325 <https://github.com/prestosql/presto/commits/325>`_
```

Comment:
is this link useful to end users?
typically https://prestosql.io/download.html is more useful

Reply:
download is already there and only maintained for latest version... we could add download links for specific release artifacts but I would rather not do that since it makes downloading old releases easier. We should not do that imho.

The tag is useful to find more technical info and be able to dig into the code if necessary and where the github issues and such are not sufficient. Imho its definitely useful.

Reply:
in my opinion, if someone is versed enough to dig into the code of a specific tag, they will have no problem is finding the tag on github by themselves 
(i assume it's not  a frequent operation anyway, so an additional hop doesn't change anything).
therefore i'd prefer not to have this link

@mosabua did this convince you?
@electrum do you have an opinion?

Reply:
Imho release notes should be as complete as possible. That should include a link to the source and in our case that is the tag. Ideally they should become even more comprehensive but I stopped at these changes for now, mostly to save time.  

I am however more than happy to remove the links again if that is the decision.

Reply:
@electrum or @martint  do you have an opinion?